lib/base.js

```diff
@@ -34,7 +34,11 @@ var Base = Class.extend({
         } else if (statusCode === 413) {
             return new AirtableError('REQUEST_TOO_LARGE', 'Request body is too large', statusCode);
         } else if (statusCode === 422) {
-            return new AirtableError(body.error.type, body.error.message, statusCode);
+            return (function(){
+                var message = (body && body.error && body.error.type) ? body.error.type : 'UNPROCESSABLE_ENTITY';
```

Comment:
Thanks for your contribution! It looks like these two lines are switched, should be:

    var message = (body && body.error && body.error.message) ? body.error.message : 'The operation cannot be processed';
    var type = (body && body.error && body.error.type) ? body.error.type : 'UNPROCESSABLE_ENTITY';

Reply:
ah yeah, thanks for pointing that out :)